tests/framework_tests/test_analytics.py

```diff
@@ -5,17 +5,20 @@
 
 import unittest
 
+import pytest
 from nose.tools import *  # flake8: noqa  (PEP8 asserts)
 from flask import Flask
 
 from datetime import datetime
 
 from framework import analytics, sessions
 from framework.sessions import session
+from osf.models import PageCounter
 
 from tests.base import OsfTestCase
 from tests.factories import UserFactory, ProjectFactory
```

Comment:
This should be changed to import from osf_tests.factories. Then [this change](https://github.com/CenterForOpenScience/osf.io/pull/6399/files#diff-bf9417725d38abe6abab2a649c39bed2R202) becomes unnecessary.